packages/repository/src/loopback-datasource-juggler.ts

```diff
@@ -95,8 +97,10 @@ export declare namespace juggler {
     static modelName: string;
     static definition: ModelDefinition;
     static attachTo(ds: DataSource): void;
-    constructor(...args: Any[]);
-    [property: string]: Any;
+    constructor(...args: any[]);
+    toJSON(): Object;
+    toObject(options?: Options): Object;
```

Comment:
I added missing `toJSON` and `toObject` from `Model` base class, so that `juggler.PersistedModel` can be used as a generic argument of `Repository<Model>`, see `packages/repository/test/unit/decorator/repository.ts` below.